tests/functional/placementapi_controller_test.go

```diff
@@ -159,6 +160,7 @@ var _ = Describe("PlacementAPI controller", func() {
 			mariadb.SimulateMariaDBDatabaseCompleted(names.MariaDBDatabaseName)
 			mariadb.SimulateMariaDBAccountCompleted(names.MariaDBAccount)
 			th.SimulateJobSuccess(names.DBSyncJobName)
+			th.SimulateDeploymentAnyNumberReplicaReady(names.DeploymentName, 0)
 			placement := GetPlacementAPI(names.PlacementAPIName)
 			Expect(*(placement.Spec.Replicas)).Should(Equal(int32(0)))
```

Comment:
and when you are at it I think this wanted to check the ReadyCount instead as the spec is set above
```golang
			Expect(placement.Status.ReadyCount).Should(Equal(int32(0)))
```

Reply:
done, but I added this check because both of them make sense